### ISSUE TITLE
Dockstore should host benchmarking dataset(s) for helping users submit cloud costs

### ISSUE THREAD

**Issue by bethsheets** [dockstore/dockstore#4101]
**Is your feature request related to a problem? Please describe.**
As users migrate to the cloud, some of the biggest barriers are cloud costs. Dockstore should host cloud cost information alongside workflows, but this may vary widely due input files, machine types, preemptibles, etc. One way to help standardize cloud cost info is a benchmarking dataset that everyone could use. This may help certain workflows be more directly comparable. For example, someone who wants to call structural variants for their cohort could consider scientific and practical (cost) considerations for their study.

**Describe the solution you'd like**
Dockstore should host example, open-access, benchmarking data (in both GCP and AWS; could set requester pays permissions) for people to use when submitting their workflows. It would be really great to host both test data and a medium-sized dataset. This could be generated from 1000 Genomes, and include CRAMs, CRAIs, multi-sample VCF, synthetic phenotype data, etc. You could ask users to submit additional data types they may need for their workflows to grow the catalog over time. I think this solution would help users submit example data in general as they don't have to generate it themselves and find a way to host it when sharing their workflows.

**Comment by denis-yuen** [comment]
To clarify, Dockstore can register example parameter files that point at data. 
Is this proposing that we also store that data ourselves and distribute it directly?

**Comment by bethsheets** [comment]
Yes, so that benchmarking data is more comparable across Dockstore workflows.
